pkg/agent/multicluster/stretched_networkpolicy_controller_test.go

```diff
@@ -0,0 +1,595 @@
+	interfacestoretest "antrea.io/antrea/pkg/agent/interfacestore/testing"
+	"antrea.io/antrea/pkg/agent/openflow"
+	oftest "antrea.io/antrea/pkg/agent/openflow/testing"
+	antreatypes "antrea.io/antrea/pkg/agent/types"
+	ovsconfigtest "antrea.io/antrea/pkg/ovs/ovsconfig/testing"
+	"antrea.io/antrea/pkg/util/channel"
+)
+
+const (
+	interval = time.Millisecond
+	timeout  = time.Second
+)
+
+type fakeStretchedNetworkPolicyController struct {
+	*StretchedNetworkPolicyController
+	clientset         *fake.Clientset
+	mcClient          *mcfake.Clientset
+	informerFactory   informers.SharedInformerFactory
+	mcInformerFactory mcinformers.SharedInformerFactory
+	ofClient          *oftest.MockClient
+	ovsClient         *ovsconfigtest.MockOVSBridgeClient
+	interfaceStore    *interfacestoretest.MockInterfaceStore
+	podUpdateChannel  *channel.SubscribableChannel
+}
+
+func newStretchedNetworkPolicyController(t *testing.T, clientset *fake.Clientset, mcClient *mcfake.Clientset) (*fakeStretchedNetworkPolicyController, func()) {
+	informerFactory := informers.NewSharedInformerFactory(clientset, 12*time.Hour)
+	listOptions := func(options *metav1.ListOptions) {
+		options.FieldSelector = fields.OneTermEqualSelector("spec.nodeName", "test-node").String()
+	}
+	podInformer := coreinformers.NewFilteredPodInformer(
+		clientset,
+		metav1.NamespaceAll,
+		0,
+		cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc}, // NamespaceIndex is used in NPLController.
+		listOptions,
+	)
+	nsInformer := informerFactory.Core().V1().Namespaces()
+	mcInformerFactory := mcinformers.NewSharedInformerFactory(mcClient, 60*time.Second)
+	labelIDInformer := mcInformerFactory.Multicluster().V1alpha1().LabelIdentities()
+
+	podUpdateChannel := channel.NewSubscribableChannel("PodUpdate", 100)
+	ctrl := gomock.NewController(t)
+	ofClient := oftest.NewMockClient(ctrl)
+	ovsClient := ovsconfigtest.NewMockOVSBridgeClient(ctrl)
+	interfaceStore := interfacestoretest.NewMockInterfaceStore(ctrl)
+	c := NewMCAgentStretchedNetworkPolicyController(
+		ofClient,
+		interfaceStore,
+		podInformer,
+		nsInformer,
+		labelIDInformer,
+		podUpdateChannel,
+	)
+	return &fakeStretchedNetworkPolicyController{
+		StretchedNetworkPolicyController: c,
+		clientset:                        clientset,
+		mcClient:                         mcClient,
+		informerFactory:                  informerFactory,
+		mcInformerFactory:                mcInformerFactory,
+		ofClient:                         ofClient,
+		ovsClient:                        ovsClient,
+		interfaceStore:                   interfaceStore,
+		podUpdateChannel:                 podUpdateChannel,
+	}, ctrl.Finish
+}
+
+var (
+	interfaceConfig = interfacestore.InterfaceConfig{
+		InterfaceName: "foo",
+		OVSPortConfig: &interfacestore.OVSPortConfig{
+			OFPort: 2,
+		},
+		IPs: []net.IP{[]byte("1.1.1.1")},
+	}
+	unknownLabelIdentity = openflow.UnknownLabelIdentity
+)
+
+func TestEnqueueAllPods(t *testing.T) {
+	var (
+		ns = &corev1.Namespace{
```

Comment:
```suggestion
	ns := &corev1.Namespace{
```
to avoid unnecessary indent

Reply:
Done.